README.md

```diff
@@ -109,29 +109,54 @@ Because Friendica links the administrator account to a specific mail address, yo
 
 ## Mail settings
 
-The binary `ssmtp` is used for the `mail()` support of Friendica.
+The binary `msmtp` is used for the mail support of Friendica.
 
-You have to set the `--hostname/-h` parameter correctly to use the right domainname for the `mail()` command.
+The mail functionality is e.g. used for sending confirmation emails for registration (including the password for newly registered
+users).
 
-You have to set a valid SMTP-MTA for the `SMTP` environment variable to enable mail support in Friendica.
-A valid SMTP-MTA would be, for example, `mx.example.org`.
+To make use of the mail functionality you need a working email account with which you can send emails. This may
+be an account on gmail, gmx or any other provider of public email. If you have your own email server you can use it as well.
+It is recommended to not use your personal email account for this. But you may use it if you just want to test friendica or during
+the installation. You can change it afterwards by simply changing the following environment variables.
```

Comment:
```suggestion
To make use of the mail functionality you need a working email account with which you can send emails.
This may be an account on gmail, gmx or any other provider of public email. If you have your own email server you can use it as well.
It is recommended to not use your personal email account for this. But you may use it if you just want to test friendica or during the installation.
You can change it afterwards by simply changing the following environment variables.
```